app/requirements.txt

```diff
@@ -2,3 +2,4 @@ Django==4.1.7
 environs==7.3.1
 urllib3==1.26.14
 setuptools==67.4.0
+Pillow==8.4.0
```

Comment:
what is pillow being used for ?

Reply:
Pillow allows to use ImageField in the model class to load images.